dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt

```diff
@@ -1427,7 +1427,7 @@ fun ChutneyStepBuilder.RadiusAuthenticateTask(
     target: String,
     userName: String,
     userPassword: String,
-    protocol: String? = "chap",
+    protocol: String? = null,
```

Comment:
Breaking change here.
We have to inform users.

Reply:
Already defaulted to chap in RadiusAuthenticateTask